src/common/components.scss

```diff
@@ -108,6 +108,95 @@
       --botsquad-bot-avatar-size: 24px;
     }
   }
+  .botsi-web-client.block-input {
+    input-method-header {
+      .controls {
+        cursor: none;
+        pointer-events: none;
+      }
+    }
+
+    .quick-replies {
+      .button {
+        cursor: none;
+        pointer-events: none;
+      }
+    }
+    .item-picker {
+      button {
+        cursor: pointer;
+        pointer-events: auto;
+      }
+    }
+    .item-picker {
+      div {
```

Comment:
you can use `div, button` to save some code duplication with the selector above this